auth/service.go

```diff
@@ -656,7 +656,7 @@ func (svc service) ChangeDomainStatus(ctx context.Context, token, id string, d D
 		ObjectType:  DomainType,
 		Permission:  AdminPermission,
 	}); err != nil {
-		return Domain{}, errors.Wrap(svcerr.ErrAuthorization, err)
+		return Domain{}, err
```

Comment:
`svc.Authorize` go#L181-L182  this function belongs to Auth service and It is exported function, 
So this function will return error with svcerr wrapped. 
Reference: https://github.com/WashingtonKK/magistrala/blob/81328c217c29766674966f63cb4b834fed586294/auth/service.
So here there is no need to warp again here `svcerr`